core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala

```diff
@@ -98,36 +98,36 @@ class CollectInformation(apps: Seq[ApplicationInfo]) extends Logging {
 
     def getIoMetrics(metricsName: String, metricsValue: String): IoMetrics = {
       metricsName match {
-        case value if (value == buffer_time) => IoMetrics(metricsValue, "", "", "")
-        case value if (value == scan_time) => IoMetrics("", metricsValue, "", "")
-        case value if (value == data_size) => IoMetrics("", "", metricsValue, "")
-        case value if (value == decode_time) => IoMetrics("", "", "", metricsValue)
+        case `buffer_time` => IoMetrics(metricsValue, "", "", "")
```

Comment:
I thought before this was comparing vs the variable buffer_time?
ie    val buffer_time: String = "buffer time"

Reply:
Earlier I was comparing `if value == buffer_time` which would translate to `if value == "buffer time"`. 
With the updated change, matching with just  `buffer_time` will treat buffer_time as variable pattern where as we need it as stable identifier pattern in this case. So backquote is required during pattern matching.
https://www.scala-lang.org/files/archive/spec/2.11/08-pattern-matching.html#stable-identifier-patterns 
Another approach is to change the variable name from `buffer_time` to `Buffer_time` and backquotes is not required then.